khmer/khmer_args.py

```diff
@@ -110,6 +272,16 @@ def __call__(self, parser, namespace, values, option_string=None):
 ** Your values for ksize, n_tables, and tablesize
 ** will be ignored.'''.format(hashfile=values))
 
+            if getattr(namespace, 'unique_kmers') != 0 or \
```

Comment:
I was just using the syntax I saw elsewhere. Referencing the attributes directly should be safe here.